src/wolfkrow/core/tasks/nuke_render.py

```diff
@@ -58,24 +79,8 @@ class NukeRender(NukeTask):
     destination = TaskAttribute(default_value="", configurable=True, attribute_type=str)
     file_type = TaskAttribute(default_value="exr", configurable=True, attribute_type=str)
     bit_depth = TaskAttribute(default_value="16 bit half", configurable=True, attribute_type=str)
-    codec = TaskAttribute(default_value=8, configurable=True, attribute_options=list(range(0, 13)), attribute_type=int, 
-        description="""
-    Which Codec to render the quicktime with.
-        0 - Apple ProRes 4444 XQ
-        1 - Apple ProRes 4444
-        2 - Apple ProRes 422 HQ
-        3 - Apple ProRes 422
-        4 - Apple ProRes 422 LT
-        5 - Apple ProRes 422 Proxy
-        6 - Avid DNxHD codec
-        7 - N/A
-        8 - Photo - JPEG
-        9 - MPEG-1 Video
-        10 - MPEG-4 Video
-        11 - PNG
-        12 - Animation
-        13 - Uncompressed 10-bit 4:2:2
-    """)
+    codec = TaskAttribute(default_value=8, configurable=True, attribute_type=str, 
+        description="""Which Codec to render the quicktime with.""")
```

Comment:
Nuke changes this stuff between versions, so lets try and find a better way of doing this... For now I think doing it based off the text value is better than the index value..

Reply:
There are other TaskAttributes in here which also use the drop down index... Lets at least be consistent and update everything.